frontend/src/pages/CollaborationPage.js

```diff
@@ -94,16 +94,7 @@ const CollaborationPage = () => {
   }, [])
 
   const getQuestion = async () => {
-    if (location.state && location.state.qnsid) {
```

Comment:
This page should really not be fetching a question based on the state passed in from the home page... the source of truth should be the question id given by matching-service anyway